test/extended/olm/olmv1.go

```diff
@@ -109,7 +109,7 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 
 		const (
 			packageName = "cluster-logging"
-			version     = "6.2.2"
+			version     = "6.3.0"
```

Comment:
Cluster Logging and its version do not exist in 4.20
We need a scenario that works for 4.20 and also for 4.19 :-( 
So that we can move forward and bump 4.20 catalogs